.releaserc

```diff
@@ -27,7 +27,7 @@
       "semantic-release-slack-bot",
       {
         "notifyOnSuccess": true,
-        "notifyOnFail": true
+        "notifyOnFail": false
```

Comment:
no need to notify on slack about failed CI runs

Reply:
we're using a slack webhook to notify about releases/CI runs (AdminJS Slack), this causes a lot of spam if there is a run which fails